guides/10_Video_Processor.md

```diff
@@ -161,3 +163,23 @@ await meetingSession.audioVideo.chooseVideoInputDevice(transformDevice);
 meetingSession.audioVideo.startLocalVideo();
 
 ```
+
+### Custom processor usage in the preview window
+
+```typescript
+import ResizeProcessor from './videofilter/ResizeProcessor';
```

Comment:
Builders may not understand importing from './videofilter/ResizeProcessor'. I see above in this guide there is an example of a class showing implementation of `VideoResizeProcessor`. Can we provide similar implementation example for `ResizeProcessor`?

Reply:
I have updated the import and naming. Can you please take a look?